src/integration/pricing/PricingHelper.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable @typescript-eslint/member-ordering */
+/* eslint-disable max-len */
+import { DimensionType, PricedConsumptionData, PricedDimensionData, PricingDimensions } from '../../types/Pricing';
+
+import Decimal from 'decimal.js';
+import Utils from '../../utils/Utils';
+
+export default class PricingHelper {
+
+  public static accumulatePricingDimensions(pricingDimensions: PricingDimensions[]): { cumulatedAmount: number, cumulatedRoundedAmount: number } {
```

Comment:
Create a type for this?
{ cumulatedAmount: number, cumulatedRoundedAmount: number }